datadog_cluster_agent/manifest.json

```diff
@@ -0,0 +1,34 @@
+{
+  "display_name": "Datadog Cluster Agent",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "datadog_cluster_agent",
+  "metric_prefix": "datadog.cluster_agent.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Track Datadog Cluster Agent's metrics",
+  "guid": "2b116278-fa9c-4efd-a250-fa6baf209607",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog Cluster Agent Integration",
+  "categories": [
+    "containers"
+  ],
+  "type": "check",
+  "is_public": true,
+  "integration_id": "datadog-cluster-agent",
```

Comment:
How come you use `datadog_cluster_agent` for `name` but then use `datadog-cluster-agent` for 'integration-id`?

Reply:
Surely an oversight. Which one would be preferred? `datadog_cluster_agent `?

Reply:
I would prefer `datadog_cluster_agent` just to be consistent with the directory name.

Reply:
Huh, I guess this is why it used the hyphen before, [it makes CI unhappy](https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=28002&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=d498e575-94b8-5995-bf82-1702cd2a023c&l=16).

```
  integration_id:'datadog_cluster_agent' does not match '^[a-z][a-z0-9-]{0,254}(?<!-)$'
```

Reply:
Interesting, I guess that it should be hyphen then. Albeit could be a bit confusing in the future.